src/components/weather-selector/weather-selector.ts

```diff
@@ -0,0 +1,63 @@
+import { Component } from '@angular/core';
+import { ViewController } from 'ionic-angular';
+@Component({
+  selector: 'weather-selector',
+  templateUrl: 'weather-selector.html'
+})
+export class WeatherSelectorComponent {
+  selectedWeather: string = '';
+  weatherConditions: object[] = [
+    { label: 'Bright roads', value: 'brightRoads' },
+    { label: 'Wet roads', value: 'WetRoads' },
+    { label: 'Dull Roads', value: 'dullRoads' },
+    { label: 'Dry Roads', value: 'dryRoads' },
+    { label: 'Raining Through Test', value: 'raining' },
+    { label: 'Snowing', value: 'snow' },
+    { label: 'Showers', value: 'showers' },
+    { label: 'Icy', value: 'icy' },
+    { label: 'Foggy/Misty', value: 'fog' },
+    { label: 'Windy', value: 'wind' }
+  ];
+
+  constructor(public viewCtrl: ViewController) {}
+
+  isEven(i) {
+    return i % 2;
+  }
+
+  updateWeatherConditions(id, isChecked) {
+    return this.weatherConditions.map((condition: any) => {
```

Comment:
this is sth that I found in my PR, tslint should catch it and warn you about using `( )` for one argument. I fixed it by setting vsc workspace settings. I added this key
```js
 "tslint.configFile": "tslint.json"
```